### ISSUE TITLE
composer install fails - case insensitive filesystem

### ISSUE THREAD

**Issue by croxton** [dukt/analytics#6]
### Description
```
ad58e4b1e2553eee0957db' -d '/Users/markcroxton/Sites/[redacted]/vendor/composer/6556a09c'

replace /Users/markcroxton/Sites/[redacted]/vendor/composer/6556a09c/google-google-api-php-client-services-26bcd71/src/Google/Service/Compute/HttpHealthCheck.php? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
(EOF or read error, treating as "[N]one" ...)

    The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
    Unzip with unzip command failed, falling back to ZipArchive class
```

### Steps to reproduce

1. when trying to install a craft site on a second mac. Works on the first (installed via Plugin store)!
2. run `composer install --no-interaction --prefer-dist --optimize-autoloader`

### Additional info

Composer version 1.6.2
Mac OS X

- Plugin version: 4.0.0-beta.2
- Craft version: 3.0.7
- PHP version: 7.0.12
- Database driver & version:
- Plugins & versions:

**Comment by croxton** [comment]
Fails to install with just the following:

```
  "require": {
    "php": ">=7.0.0",
    "craftcms/cms": "3.0.7",
    "dukt/analytics": "4.0.0-beta.2"
  },
```

**Comment by benjamindavid** [comment]
Should be fixed by [4.0.0-beta.3](https://github.com/dukt/analytics/blob/v4/CHANGELOG.md#400-beta3---2018-05-16). Thanks for reporting!
